tests/test_rms_config.py

```diff
@@ -210,7 +210,7 @@ def test_rmsconfig_get_env(
         ("14.2.1", "V14.2", "14.2.2"),
         ("14.2.2", "V14.2", "14.2.2"),
         ("14.2.1", "V14.2", "14.2.2"),
-        ("14.5.0", "V14.5", "14.5.0"),
+        ("14.5", "V14.5", "14.5"),
```

Comment:
Small note: "V14.5" is also equal to "14.5.0.0" in the "About" info from RMS, but not sure if that is relevant here.